blog/_posts/2018-07-07-brazilian-team.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Brazilian Housewives Coding Team Introduction
+layout: post
+created_at: Mon July 09 2018
+permalink: blog/2018-07-07-brazilian-team
+current: blog
+author: Camila and Stefanni
+twitter: milasouzaaraujo
+categories:
+- student-posts
+---
+
+### About us
+
+Hello! We are [Stefanni](https://twitter.com/steff_br) and [Camila](https://twitter.com/milasouzaaraujo), two junior software developers from Brazil based in Vancouver, Canada. We didn't know each other before the RGSoC and we are more than happy to introduce our team!
+
+#### [Stefanni](http://stefannibrasil.me/)
+
+I am a millennial that likes to how to cook (gasp) and loves coding, cats, playing sports and chocolate. I moved to Vancouver in January because my husband received a job offer to work here and we brought our two cats with us.
```

Comment:
Hi @milaaraujo ! Thanks for the PR. See the comments below.

The sentence needs a little bit modification. '... am a millennial that likes how to cook, loves coding, cats ...'.